supervisor/resolution/evaluations/docker_configuration.py

```diff
@@ -52,13 +53,17 @@ async def evaluate(self):
                 "Docker logging driver %s is not supported!", logging_driver
             )
 
-        if cgroup_version != EXPECTED_CGROUP_VERSION:
+        expected_version = [CGROUP_V1_VERSION]
+        if self.coresys.os.available:
```

Comment:
It seems that this property is not initialized yet when this is evaluated. Any idea how this can be fixed?